Description: PAN GlobalProtect: Add gp to supported-protocols list
Author: Sergi Granell <xerpi.g.12@gmail.com>
Origin: upstream, https://gitlab.gnome.org/GNOME/NetworkManager-openconnect/commit/296bf10eebfc3423b4e03215a84cd7ec7ef10e16
Bug: https://gitlab.gnome.org/GNOME/NetworkManager-openconnect/issues/1
Bug-Debian: https://bugs.debian.org/925081
Last-Update: 2019-05-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/nm-openconnect-service.name.in
+++ b/nm-openconnect-service.name.in
@@ -12,4 +12,4 @@
 properties=@PLUGINDIR@/libnm-openconnect-properties
 
 [openconnect]
-supported-protocols=anyconnect,nc
+supported-protocols=anyconnect,nc,gp
